src/environment.js

```diff
@@ -289,9 +299,11 @@ export default class Environment extends EventEmitter {
    * Resolve given file from `this.dir`.
    *
    * @param {String} file
+   * @param {Boolean} cwd - whether it's relative to CWD (like when
+   *                        defined in CLI).
    * @return {String}
    */
-  resolve(file) {
-    return path.resolve(this.dir, file);
+  resolve(file, cwd = false) {
```

Comment:
Similarly here, I would rather have a "from CLI" or "CLI enabled" naming, rather than "themeCli" and "destCLI" which may lead to think there are 2 different things.